org.kde.kommit.json

```diff
@@ -0,0 +1,100 @@
+{
+    "id": "org.kde.kommit",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kommit",
+    "rename-icon": "kommit",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=ssh-auth",
+        "--socket=gpg-agent",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--filesystem=home",
+        "--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "git",
+            "make-args": [
+                "INSTALL_SYMLINKS=1"
+            ],
+            "make-install-args": [
+                "INSTALL_SYMLINKS=1"
+            ],
+            "cleanup": [
+                "/lib/pkgconfig",
+                "/man"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.kernel.org/pub/software/scm/git/git-2.41.0.tar.xz",
+                    "sha256": "e748bafd424cfe80b212cbc6f1bbccc3a47d4862fb1eb7988877750478568040",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 5350,
+                        "stable-only": true,
+                        "url-template": "https://www.kernel.org/pub/software/scm/git/git-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "git-lfs",
+            "buildsystem": "simple",
+            "build-commands": [
+                "PREFIX=${FLATPAK_DEST} ./install.sh"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "strip-components": 1,
+                    "url": "https://github.com/git-lfs/git-lfs/releases/download/v3.4.0/git-lfs-linux-amd64-v3.4.0.tar.gz",
```

Comment:
Why are we downloading a binary instead of compiling it?

Reply:
Ask the authors of https://github.com/flathub/org.flatpak.Builder/blob/master/org.flatpak.Builder.json#L238 I assume packaging Go is not well supported yet.

Reply:
I don't know why I would ask people unrelated to this MR, they are not proposing anything.

https://github.com/fyne-io/flatpak_demo/blob/main/io.fyne.flatpak_demo.yml seems to say building a go app just works.

Reply:
I mean, they probably had their reasons to choose that path in an official @flathub package.

Reply:
What do you mean "an official flathub package"?

Reply:
My understanding is that git-lfs had botched vendoring the Go packages, so build kept failing in the pursuit of always having the latest package. (This is not an approval)

`org.flatpak.Builder` is maintained by the Flathub maintainers as it is the one used on the builders.

Reply:
> org.flatpak.Builder is maintained by the Flathub maintainers as it is the one used on the builders.

This is starting to discussing the smell of clours, but that doesn't really make it a more official package than say Okular.

> My understanding is that git-lfs had botched vendoring the Go packages, so build kept failing in the pursuit of always having the latest package. (This is not an approval)

I would really like if there was an attempt at making this work and someone could show me the error before we accept defeat and include a "random binary" in this package.